staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/applyconfiguration/cr/v1/example.go

```diff
@@ -218,8 +219,24 @@ func (b *ExampleApplyConfiguration) WithStatus(value *ExampleStatusApplyConfigur
 	return b
 }
 
+// GetKind retrieves the value of the Kind field in the declarative configuration.
+func (b *ExampleApplyConfiguration) GetKind() *string {
+	return b.TypeMetaApplyConfiguration.Kind
+}
+
+// GetAPIVersion retrieves the value of the APIVersion field in the declarative configuration.
+func (b *ExampleApplyConfiguration) GetAPIVersion() *string {
+	return b.TypeMetaApplyConfiguration.APIVersion
+}
+
```

Comment:
I was confused why these methods were generated, but eventually I figured that now `applyconfiguration.go:needsGetter()` is returning `true` for this type.